smoke_zephyr/configuration.py

```diff
@@ -202,6 +202,63 @@ def save(self):
 		with open(self.configuration_file, 'w') as file_h:
 			file_h.write(self._serializer('dumps', self._storage))
 
+class MemoryConfiguration(Configuration):
```

Comment:
This should be the opposite. `MemoryConfiguration` should be the parent class, and `Configuraiton` should inherit from it.